catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1273,7 +1274,9 @@ public UpdateResponse update(UpdateRequest updateRequest)
             queryImpl.setStartIndex(1);
             queryImpl.setPageSize(updateReq.getUpdates()
                     .size());
-            QueryRequestImpl queryRequest = new QueryRequestImpl(queryImpl);
+            QueryRequestImpl queryRequest = new QueryRequestImpl(queryImpl,
+                    updateReq.getStoreIds());
```

Comment:
Will the request always have at least 1 storeId? I don't think there is anything enforcing that. We should probably check, and assume local provider if it's empty.